.github/workflows/push-dockerhub-tag.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and push tagged release to Docker Hub
+
+on:
+  push:
+    tags:
+      - 'v*.*'
+      - 'latest'
```

Comment:
The tag version should work, I havent tested if `latest` will work, at the worst it will just require duplicating this recipe and replacing the tag with the string 'latest' on line 58

also supports multiple platforms dockerx build recipe